index.html

```diff
@@ -30,7 +31,7 @@
                 to live, work or just relax
             </p>
         </div>
-        <div class="header-search">
+        <form class="header-search">
             <div class="search-input">
                 <div class="input">
                     <input type="search" name="search-input" placeholder="Your destination or hotel name">
```

Comment:
it's not a placeholder - maybe use default value equals to "Your destination or hotel name"